cli.js

```diff
@@ -11,11 +11,10 @@ meow({
 	]
 });
 
-fullname(function (err, name) {
-	if (err) {
+fullname()
+	.then(function (name) {
+		console.log(name);
+	}).catch(function (err) {
```

Comment:
`meow` handles the catching of promises. Just bump it to latest version: `3.4.0`